src/encoded/static/components/item-pages/components/CollapsibleItemViewButtonToolbar.js

```diff
@@ -56,23 +54,29 @@ export class CollapsibleItemViewButtonToolbar extends React.PureComponent {
             );
         }
 
-        var { children, windowWidth, collapseButtonTitle } = this.props,
+        var { children, windowWidth, collapseButtonTitle, session } = this.props,
             gridState       = this.state.mounted && layout.responsiveGridState(windowWidth),
             isMobileSize    = gridState && gridState !== 'lg',
             isOpen          = !isMobileSize || this.state.open;
 
+        // If the user isn't logged in, add a tooltip reminding them to log in.
+        var tooltip = null;
+        if (!session) {
```

Comment:
CollapsibleItemViewButtonToolbar.js is meant to be re-usable and not tied to HiGlass specifically so that we could re-use it on other Item views that might need a toolbar in their title area later on, e.g. maybe WorkflowRuns at some point.

Would move this `if (!session){...` logic up into `HiGlassViewConfigTabView` and instead have `CollapsibleItemViewButtonToolbar` accept a more-ambiguous `data-tip` (or `tooltip`, w/e name) prop that then gets applied to the `Collapse`/`ButtonToolbar`.